eta-meeting-organizer-backend/src/main/java/hu/flowacademy/meetingorganizer/service/UserService.java

```diff
@@ -37,9 +37,14 @@ public void deleteUser(String id) {
     userRepository.deleteById(id);
   }
 
-  public User updateUser(String id) {
+  public User updateUser(String id, User user) {
+    user.setId(id);
+    return userRepository.save(user);
+  }
+
+  public User setUserRole(String id, RoleDTO roleDTO) {
     User user = userRepository.findById(id).orElseThrow();
-    user.setRole(Role.USER);
+    user.setRole(roleDTO.getRole());
     emailService.send(user.getUsername(), "validation", EmailType.TEXT);
```

Comment:
Ez ide minek kell?

Reply:
Frontendről kapott role-t ebből kapom vissza.

Reply:
Oké, pontosítok. Az e-mail küldés minek kell?

Reply:
Amikor frontend-en az admin kiválasztja, hogy a függőben lévő felhasználó milyen szerepkörrel rendelkezik, a rendszer küld egy értesítő email-t, hogy mostmár betud lépni a felhasználó. Szerepkör beállításig csak egy "hiba" üzenet fogadja, amikor megpróbál belépni.

Reply:
Á, értelek. Oké